buildscripts/kokoro/macos.cfg

```diff
@@ -15,6 +15,6 @@ env_vars {
 # We always build mvn artifacts.
 action {
   define_artifacts {
-    regex: ["**/mvn-artifacts/**"]
+    regex: "github/grpc-java/mvn-artifacts/**"
```

Comment:
This change is just to be consistent with linux_artifacts.cfg. Ditto for windows.cfg. The `[]` syntax is poor for these long strings.